kraken-build/src/kraken/std/python/buildsystem/uv.py

```diff
@@ -230,34 +228,10 @@ def requires_login(self) -> bool:
     # TODO: Implement bump_version()
 
     def build_v2(self, settings: PythonSettings, output_directory: Path) -> list[Path]:
-        """
-        Uses [build] `>=1.0.0,<2.0.0` to build a distribution of the Python project.
-
-        [build]: https://pypi.org/project/build/
-        """
-
-        with tempfile.TemporaryDirectory() as tempdir:
-            command = [
-                "uv",
-                "tool",
-                "run",
-                "--from",
-                "build>=1.0.0,<2.0.0",
-                "pyproject-build",
-                "-v",
-                "--outdir",
-                tempdir,
-                "--installer",
-                "uv",
-            ]
-            _run_uv_command_with_indexes(command, settings, self.project_directory)
-
-            src_files = list(Path(tempdir).iterdir())
-            dst_files = [output_directory / path.name for path in src_files]
-            for src, dst in zip(src_files, dst_files, strict=True):
-                shutil.move(str(src), dst)
-
-        return dst_files
+        _run_uv_command_with_indexes(
+            ["uv", "build", f"--out-dir={output_directory.absolute()}"], settings, self.project_directory
```

Comment:
@Tpt @vsiles `uv build` support multiple backends: `hatch`, `flit`, `pdm`, `setuptools` and `maturin` and `scikit`. Do you think using `uv build` here would remove the need for `MaturinUvPythonBuildSystem`? If that is the case, we would have a unified interface for all things `uv` independently of the build backend.

Reply:
Sadly not: `MaturinUvPythonBuildSystem` takes care of building wheels targetting Python stable ABI (abi3) for multiple platforms using cross-compilation whereas `maturin` called via `uv build` will by default only build a wheel for the current system.

Reply:
Agreed. The main goal here is cross-compilation. I don't think uv is providing anything like that out of the box (nor that it should)